### PR TITLE
Added mdanse_job application for running a job through a stand-alone dialog

### PR DESCRIPTION
**Pull request by eurydice76**
See issue #62 for details

### CODE REVIEW

File: Doc/conf_api.py [hunk 1]
```diff
@@ -30,14 +30,18 @@
...
+
 # -- General configuration -----------------------------------------------------
 
 # If your documentation needs a minimal Sphinx version, state it here.
 needs_sphinx = '1.0'
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.viewcode', 'sphinx.ext.graphviz', 'sphinx.ext.inheritance_diagram', 'sphinx.ext.pngmath']#,'rst2pdf.pdfbuilder']
+extensions = ['sphinx.ext.autodoc', 'sphinx.ext.viewcode', 'sphinx.ext.graphviz', 'sphinx.ext.inheritance_diagram''sphinx.ext.pngmath' if minor <= 6 else 'sphinx.ext.imgmath']
```
Comment (RastislavTuranyi) [inline Doc/conf_api.py:44]: I think you are missing a comma here. Regardless, your changes caused a lot of warnings that it cannot create inheritance diagrams when running `build_api`.

**Review by RastislavTuranyi — CHANGES_REQUESTED** [review]
The script works well, but I believe improving the information given by -h to be very useful.

Other than that, I am not sure how the changes to Doc are related to this script.

**Comment by eurydice76** [comment]
Thanks for the comments. I added more info to the `mdanse_job` script help. Is it OK for you now ? For the fix in the 
documentation, you are fully right I should have done this in a separate branch/PR. I can do it if you want

**Review by RastislavTuranyi — COMMENTED** [review]
I think it would be best make the changes to Doc in a separate branch and PR, so that we can discuss them separately. 

Thank you for making the usage of the script easier. I would say that currently just running -h gives enough info for users to be able to use the script.

**Comment by eurydice76** [comment]
OK, let's create another PR for the doc changes

**Comment by sanghamitra-mukhopadhyay** [comment]
Can we resolve the conflict ?

**Review by sanghamitra-mukhopadhyay — APPROVED** [review]

**Comment by eurydice76** [comment]
The functional tests are failing. So it becomes urgent for the branch #66 #56 and  to me merged.

**Comment by sanghamitra-mukhopadhyay** [comment]
I see a conflict in #66 and #56/ #59 are still in work in progress?

**Pull request merged**